addons/website_sale/tests/test_website_sequence.py

```diff
@@ -0,0 +1,70 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestWebsiteSequence(odoo.tests.TransactionCase):
+
+    def setUp(self):
+        super(TestWebsiteSequence, self).setUp()
+
+        ProductTemplate = self.env['product.template']
+        product_templates = ProductTemplate.search([])
+        # if stock is installed we can't archive since there is orderpoints
+        if hasattr(self.env['product.product'], 'orderpoint_ids'):
```

Comment:
@JKE-be If you have a better idea (except removing tests) I'd like to remove that crappy part

Reply:
agreed